src/modules/transport-mode/icon/index.tsx

```diff
@@ -97,7 +97,7 @@ const TRANSPORT_SUB_MODES_BOAT: TransportSubmodeType[] = [
   'sightseeingService',
 ];
 
-function modeToColor(
+export function modeToColor(
```

Comment:
Should this be renamed to e.g. `transportModeToColor` now that it is used outside this module, to be more descriptive?